notebooks/stress-tests.ipynb

```diff
@@ -38,8 +38,8 @@
      "name": "stdout",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.        :param target_fonction: Fonction to be executed by the threads
</pre><p>Param doc (target_function, args) out of sync with function definition<code>def run_threads(target_test, url, execution_id:int, n_threads:int, runs_per_threads:int)</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/PAVICS-e2e-workflow-tests/pull/90/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>